### ISSUE TITLE
Trait `VectorElement` is not implemented for `Vector<Point2d>`

### ISSUE THREAD

**Issue by wprzytula** [twistedfall/opencv-rust#514]
There is not reason for `Vector<Point2f>` implementing `VectorElement` and `Vector<Point2d>` not.

**Comment by twistedfall** [comment]
You're right in general, but those types are being generated based on their actual use in OpenCV function signatures. Sometimes it can be that a function accepts those types via `InputArray`, then a manual intervention is needed. Can you please tell me which function requires `Vector<Point2d>` in your case?

**Comment by wprzytula** [comment]
The problem I encounter is while using `opencv::ccalib::calibrate`. No matter how hard I'm trying, I'm still triggering various assertion errors. The last one is:
```
Error: OpenCV(4.8.1) /usr/src/debug/opencv/opencv_contrib-4.8.1/modules/ccalib/src/omnidir.cpp:854: error: (-215:Assertion failed) !objectPoints.empty() && objectPoints.type() == CV_64FC3 in function 'computeJacobian'
```
So I tried to change types held in `object_points` and `image_points` from `f32` to `f64`, but then I could not create neither of them, because I can't create a `Vector` of `Vector` of `Point2/3d`, because `Vector<Point2d>` does not implement `VectorElement`.

**Comment by wprzytula** [comment]
Could you by the way help me with figuring out the proper types for `image_points` and `object_points` for `opencv::ccalib::calibrate`?

**Comment by wprzytula** [comment]
Now I noticed that they are using `Vec2f`, not `Point2f`, in their tutorials. Too bad, because `Vec2f` does not implement `VectorElement`, too...

**Comment by twistedfall** [comment]
Can you please try with the `beta` branch? I have added some forced generation of `Vector` bindings for the `ccalib` module.

Additionally if I'm reading the documentation right (https://docs.opencv.org/4.x/d3/ddc/group__ccalib.html#gaf285e757a4091bfbc2bb742ada3ccba7) `objectPoints` is `Vector<Vector<Point3f/3d>>` and `imagePoints` is `Vector<Vector<Point2f>>`, note the additional `Vector` hierarchy.

**Comment by twistedfall** [comment]
This was also released as part of v0.88.0 if you still have the issue using this version please reopen

**Comment by wprzytula** [comment]
Using v0.88, I'm still getting this:
```
error[E0277]: the trait bound `opencv::core::Vector<Point_<f64>>: VectorElement` is not satisfied
   --> src/bin/hw1.rs:315:56
    |
315 |         pub fn corners_for_image(&self, img: usize) -> Vector<Vector<Point2d>> {
    |                                                        ^^^^^^^^^^^^^^^^^^^^^^^ the trait `VectorElement` is not implemented for `opencv::core::Vector<Point_<f64>>`
 ```

**Comment by twistedfall** [comment]
This should now be fixed in 0.88.1
